CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * **BREAKING CHANGE:** removed old `expand` function
 * Added `assert-key` to replace `pre-init-spec`
 * Added `expand-key` and new `expand` function (#102)
+* Added babashka test runner (#86)
```

Comment:
Can you remove this line? I typically update the changelog when I have a release.